ui/src/app/utils/cdr-versions.tsx

```diff
@@ -1,19 +1,39 @@
-import {CdrVersion, CdrVersionListResponse, Workspace} from 'generated/fetch';
+import {CdrVersion, CdrVersionTier, CdrVersionTiersResponse, Workspace} from 'generated/fetch';
+import * as fp from 'lodash/fp';
 
-function getCdrVersion(workspace: Workspace, cdrList: CdrVersionListResponse): CdrVersion {
-  return cdrList.items.find(v => v.cdrVersionId === workspace.cdrVersionId);
+function getCdrVersionTier(accessTierShortName: string, cdrTiers: CdrVersionTiersResponse): CdrVersionTier {
+  return cdrTiers.tiers.find(v => v.accessTierShortName === accessTierShortName);
 }
 
-function getDefaultCdrVersion(cdrList: CdrVersionListResponse): CdrVersion {
-  return cdrList.items.find(v => v.cdrVersionId === cdrList.defaultCdrVersionId);
+function getCdrVersion(workspace: Workspace, cdrTiers: CdrVersionTiersResponse): CdrVersion {
+  const tier = getCdrVersionTier(workspace.accessTierShortName, cdrTiers);
```

Comment:
nit: this method could arguably be more simply implemented as `findCdrVersion(workspace.cdrVersion)`. It's not strictly necessary to check the tier when doing this lookup.

Alternatively, this may also be simple enough that callers can just use that instead of having a helper.